packages/turf-point-grid/index.d.ts

```diff
@@ -3,6 +3,6 @@ import {BBox, Points, Units, Feature, Features} from '@turf/helpers';
 /**
  * http://turfjs.org/docs/#pointgrid
  */
-declare function pointGrid(bbox: BBox | Feature<any> | Features<any>, cellSide: number, units?: Units, centered?: boolean, bboxIsMask?: boolean): Points;
+declare function pointGrid(bbox: BBox | Feature<any> | Array<number>, cellSide: number, units?: Units, centered?: boolean, bboxIsMask?: boolean): Points;
```

Comment:
@DenisCarriere it'd be nice if we had some kind of `types` file (`TYPES.md`?) maybe in the root as reference for all Turf modules. With that I could help updating all the modules and would be useful, when creating new models, for who like me doesn't know typescript